index.html

```diff
@@ -281,5 +270,47 @@ <h6 class="body-1 bolder flex-text">Skills</h6>
         </li>
       </ul>
     </section>
+    <footer>
+      <section class="contact-me">
+        <h2 class="header-2 color-n0 mb-12">Contact me</h2>
+        <p class="body-4 align-center color-n0 mb-12">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/xdoygqql" method="POST">
+          <input
+            type="text"
+            name="user_name"
+            id="user_name"
+            placeholder="Type your name"
+            class="text-field"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="user_email"
+            id="user_email"
+            placeholder="Type your email"
+            class="text-field"
+            required
+          />
+          <textarea
+            name="message"
+            id="message"
+            cols="30"
+            rows="8"
+            placeholder="Write your message"
+            class="text-field"
+            maxlength="500"
+            required
+          ></textarea>
+          <div>
+            <button type="submit" class="button">Get In Touch</button>
+          </div>
+        </form>
+      </section>
```

Comment:
Still on sticking to the design templates in Figma as much as possible, kindly observe the `.gif` image below: 

![Peek 2021-06-08 12-46](https://user-images.githubusercontent.com/26856491/121179538-c8b5ce00-c857-11eb-9681-81dc36b49190.gif)

If you observe closely, you'll realize the following:
- You still have an overflow of the form fields as the screen becomes smaller.
- This overflow leads to the display of the horizontal scroll bar
To fix this, you might consider setting the width of the text fields to automatically fit the container (i.e. content), or otherwise. Kindly find a way to fix this.

**As this is your second review, kindly ensure that the required change is made before requesting for another review.**

Reply:
Is it okay to use CSS 'grid' for some layout styling?

Reply:
Hi @usmansbk,

For this project, CSS flex is recommended. I think you can still make it work with CSS flex.

Reply:
Plus this is really a lot of pressure on me having one review left. You should have added this in the first review.